### ISSUE TITLE
loading with paths

### ISSUE THREAD

**Issue by behinger** [ReneSkukies/UnfoldBIDS.jl#46]
I got lots of errors with loading in more complex environments

E.g. it tries to read empty folders, ".git" folders, it throws errors if a file has no extension either :S

---
there is also quite some code doubling, I dont want to get too knee deep in there, but once the code doubling is addressed, this becomes easier to debug?

**Comment by behinger** [comment]
ah sorry - ok, I understand now what happened. I had to put the ".git" in the excludeFolder; maybe a good idea to exclude all secret ".secret" folders by defaul?

**Comment by ReneSkukies** [comment]
Yes, we should think about this. Becomes quite the conditional statement already though...

**Comment by ReneSkukies** [comment]
I can have a look at how BIDS.jl used to do it and revise a bit

**Comment by ReneSkukies** [comment]
Fixedw with continuables
